test/script.jl

```diff
@@ -22,8 +22,19 @@ end
 sort!(hosts)
 println(hosts)
 
-# We don't use `@assert` here, for reason described above.
-if hosts != ["c1", "c1", "c2", "c2"]
-  msg = "Test failed: observed_hosts=$(hosts) does not match expected_hosts=[c1, c1, c2, c2]"
-  error(msg)
+const is_ci = parse(Bool, strip(get(ENV, "CI", "false")))
```

Comment:
Is this actually present in our CI since we are running inside the docker compose cluster?

Reply:
Ah, that's a really good point. No, I don't think it is.

Hmmm. I'll have to figure something else out.

Reply:
Alright, I have a different approach now.

If the environment variable is not set (e.g. if you are inside of our Docker Compose cluster), we default to `is_ci = true`.

Reply:
So if you look at the CI logs on this PR (e.g. [this CI log](https://github.com/kleinhenz/SlurmClusterManager.jl/actions/runs/12877371694/job/35901709965?pr=29)), you'll see the following message:

> [ Info: This is CI, so we will perform the hostname test

This indicates that the test is being run on our CI, as desired.